src/lib/interactionManager/credentialOfferFlow.ts

```diff
@@ -72,6 +75,10 @@ export class CredentialOfferFlow extends Flow<
       }
     })
 
+    const validArr = this.state.credentialsValidity = await JolocomLib.util.validateDigestables(
+      signedCredentials,
+    )
+    this.state.credentialsAllValid = validArr.reduce((prev, cur) => prev && cur, true)
```

Comment:
Super nitpick -> `validArr.every(el => el == true [or !!el])`

Reply:
Also, I believe this is the only time we are doing async stuff inside of flows (i.e. not sure if signature validation belongs here). There's no cleaner alternative (what we had before was arguably worse).

Reply:
Good catch :D it's `every(v => v)` now because typescript swears they are already booleans